crates/matrix-sdk-crypto-nodejs/src/identifiers.rs

```diff
@@ -43,13 +46,13 @@ impl UserId {
     /// A historical user ID is one that doesn't conform to the latest
     /// specification of the user ID grammar but is still accepted
     /// because it was previously allowed.
-    #[napi(getter, js_name = "isHistorical")]
```

Comment:
Why are these `js_name`s removed (and the getter attribute here, too)?

Reply:
`napi-rs` automatically transforms `foo_bar` to `fooBar`, so using all those `js_name`s is now useless.

Also, about the `getter` being removed, I don't feel like `is*` functions are getters, but more real functions semantically speaking.